chpl/chpl-resources/src/main/resources/lookup.properties

```diff
@@ -38,15 +38,24 @@ criterion.170_315_b_9_revised=170
 # 170.315(c)(3) Clinical Quality Measures - Report
 criterion.170_315_c_3=27
 criterion.170_315_c_3_revised=172
+# 170.315(d)(2) Auditable Events and Tamper-Resistance
+criterion.170_315_d_2=30
+criterion.170_315_d_2_revised=183
```

Comment:
I feel like this one should be 179, d3_revised will be 180, and d1_revised will be 181

Reply:
Could be completely wrong, but these revised ones will be inserted before the revised ones that start with any letter after d, so should have IDs lower than those

Reply:
Ended up being this when cross-referenced with prod DB and running add job on it - updated as such:
# 170.315(d)(2) Auditable Events and Tamper-Resistance
criterion.170_315_d_2=30
criterion.170_315_d_2_revised=173
# 170.315(d)(3) Audit Report(s)
criterion.170_315_d_3=31
criterion.170_315_d_3_revised=174
# 170.315(d)(10) Auditing Actions on Health Information
criterion.170_315_d_10=38
criterion.170_315_d_10_revised=175